browser-support.html

```diff
@@ -20,6 +20,7 @@
       <ul>
         <li>Google Chrome (latest stable)</li>
         <li>Apple Safari (latest)</li>
+        <li>Firefox(latest)</li>
```

Comment:
needs a space between `Firefox` and `(latest)`

I'd also suggest copying the Chrome model and saying "latest stable" since there are many tracks for Firefox